mmv1/products/apigee/EnvironmentKeyvaluemap.yaml

```diff
@@ -0,0 +1,82 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'EnvironmentKeyvaluemaps'
+description: |
+  Collection of key/value string pairs.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Using key value maps': 'https://cloud.google.com/apigee/docs/api-platform/cache/key-value-maps'
+  api: 'https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.environments.keyvaluemaps/create'
+
+base_url: '{{env_id}}/keyvaluemaps'
+create_url: '{{env_id}}/keyvaluemaps'
+delete_url: '{{env_id}}/keyvaluemaps/{{name}}'
+self_link: '{{env_id}}/keyvaluemaps/{{name}}/entries'
+import_format: ['{{env_id}}/keyvaluemaps/{{name}}', '{{env_id}}/{{name}}']
+skip_sweeper: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'apigee_environment_keyvaluemaps_basic'
+    primary_resource_id: 'apigee_environment_keyvaluemaps'
+    test_env_vars:
+      org_id: :ORG_ID
+      billing_account: :BILLING_ACCT
+    skip_test: true
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'apigee_environment_keyvaluemaps_test'
+    primary_resource_id: 'apigee_environment_keyvaluemaps'
+    test_env_vars:
+      org_id: :ORG_ID
+      billing_account: :BILLING_ACCT
+    skip_docs: true
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'apigee_environment_keyvaluemaps_beta_test'
+    primary_resource_id: 'apigee_environment_keyvaluemaps'
+    test_env_vars:
+      org_id: :ORG_ID
+      billing_account: :BILLING_ACCT
+    skip_docs: true
+    min_version: beta
+immutable: true
+timeouts: !ruby/object:Api::Timeouts
+  insert_minutes: 1
+  delete_minutes: 1
+autogen_async: true
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'envId'
+    description: |
+      The Apigee environment group associated with the Apigee environment,
+      in the format `organizations/{{org_name}}/environments/{{env_name}}`.
+    required: true
+    immutable: true
+    url_param_only: true
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  custom_import: templates/terraform/custom_import/apigee_environment_keyvaluemaps.go.erb
+  custom_create: 'templates/terraform/custom_create/apigee_environment_keyvaluemaps.go'
+  decoder: templates/terraform/decoders/apigee_environment_keyvaluemaps.go.erb
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    description: |
+      Required. ID of the key value map.
+    required: true
+    immutable: true
+  - !ruby/object:Api::Type::Boolean
+    name: 'encrypted'
```

Comment:
It does seem that this field is always needs to be `true`. Then should we omit this field and just send `encrypted:true` in the API call? What do you think?

Reply:
I think we can omit the field in this case, given that the field is retained for backward compatibility in the API and only fixed value is accepted.

Reply:
change checked-in.